pages/policy.html

```diff
@@ -5,6 +5,29 @@
 ---
 
 <div id="section-full">
+    <div class="section-header">
+        <h3>Gameplay streaming, Let's Play content, and Monetization Guidelines</h3>
+    </div>
+    <div class="section-item">
+        <div class="content">
+            <p>The core game of OpenTTD is released under the GNU General Public License (GPL) version 2.</p>
+            <p>The OpenGFX baseset graphics pack is released under the GNU General Public License version 2.</p>
+            <p>The OpenSFX baseset sound effects pack is released under the Creative Commons Attribution-ShareAlike 3.0 Unported license.</p>
+            <p>The OpenMSX baseset music pack is released under the GNU General Public License version 2.</p>
+            <p>All the above licenses permit use of the software for any purpose, including commercial purposes.</p>
+            <p>Following is a statement of intention from the developers. This is not a legal statement. If any questions arise from this statement, please consult the license text of the involved software and game data.</p>
+            <h3>General permission for the unmodified game</h3>
+            <p>It is the intention of the development teams behind OpenTTD and the open baseset game data packs that any Let's Play, review, and other similar content, is permitted with no limitations. Monetization and commercial use of the game and game data is permitted without limitations.</p>
+            <p>Note that regardless of this statement, any distribution of the game and game data files still falls under the restrictions of the respective licenses.</p>
+            <p>Note that this statement does not imply any endorsement from any of the developers of OpenTTD or the baseset game data towards any content creators. Any views expressed by content creators in content using the OpenTTD game and baseset game data packs are purely those of the content creator.</p>
+            <h3>Non-coverage of use of other baseset data</h3>
```

Comment:
Nitpick: `of use of other` -> `of use with` (similar as with `add-on game data`. `of .. of` reads weird :)